service-front/module/Application/src/Helpers/LpaFormHelper.php

```diff
@@ -0,0 +1,220 @@
+            }
+            if (! $response['address_match'] || ! $response['name_match']) {
+                $response['error'] = true;
+            }
+        } catch (\Exception $exception) {
+            $response['error'] = true;
+            $response['info'] = $exception->getMessage();
+        }
+        return $response;
+    }
+
+    private function checkStatus(array $siriusCheck): array
+    {
+        $response = [
+            'error' => false,
+            'status' => "",
+            'message' => ""
+        ];
+        if (
+            array_key_exists('opg.poas.lpastore', $siriusCheck) &&
+            array_key_exists('status', $siriusCheck['opg.poas.lpastore'])
+        ) {
+            $response['status'] = $siriusCheck['opg.poas.lpastore']['status'];
+            if (
+                $response['status'] == 'complete' ||
+                $response['status'] == 'registered' ||
+                $response['status'] == 'in progress'
+            ) {
+                $response['error'] = true;
+                $response['message'] = "This LPA cannot be added as an ID" .
+                    " check has already been completed for this LPA.";
+            }
+            if ($response['status'] == 'draft') {
+                $response['error'] = true;
+                $response['message'] = "This LPA cannot be added as it’s status is set to Draft.
+                    LPAs need to be in the In Progress status to be added to this ID check.";
+            }
+        } else {
+            $response['error'] = true;
+            $response['message'] = "No LPA Found.";
+        }
+        return $response;
+    }
+
+    private function checkChannel(array $siriusCheck): array
+    {
+        $response = [];
+        $response['channel'] = 'paper';
+        $response['error'] = false;
+        $response['message'] = "";
+        $response['status'] = $siriusCheck['opg.poas.lpastore']['status'];
+
+        if (
+            array_key_exists('opg.poas.lpastore', $siriusCheck) &&
+            array_key_exists('status', $siriusCheck['opg.poas.lpastore'])
+        ) {
+            $response['channel'] = $siriusCheck['opg.poas.lpastore']['certificateProvider']['channel'];
+            if ($response['channel'] == 'online') {
+                $response['error'] = true;
+                $response['message'] = "This LPA cannot be added to this identity check because
+                    the certificate provider has signed this LPA online.";
+            }
+        }
+        return $response;
+    }
+
+    private function checkLpaNotAdded(string $lpa, array $detailsData): bool
+    {
+        try {
+            foreach ($detailsData['lpas'] as $existingLpa) {
+                if ($lpa == $existingLpa) {
+                    return false;
+                }
+            }
+            return true;
+        } catch (\Exception $exception) {
+            return false;
+        }
```

Comment:
Sorry, just spotted that this also has an unnecessary try..catch statement (since there's nothing in the `try` that could throw an exception)
```suggestion
        foreach ($detailsData['lpas'] as $existingLpa) {
            if ($lpa == $existingLpa) {
                return false;
            }
        }

        return true;
```